src/sdk-reference/cpp/1/auth/check-token/index.md

```diff
@@ -3,7 +3,6 @@ layout: sdk.html.hbs
 algolia: true
 title: checkToken
 description: Checks a JWT Token's validity.
-order: 200
 ---
```

Comment:
It has no use: the `order` parameter should only be used to force items to appear before others. 
Otherwise the default behavior should be applied: let the items be alphabetically sorted.

This behavior comes directly from the 1st version of our documentation.